docs/en/connector-v2/sink/StarRocks.md

```diff
@@ -2,94 +2,44 @@
+## Description
 
-Whether to enable upsert/delete, only supports PrimaryKey model.
+Used to send data to StarRocks. Both support streaming and batch mode.
+The internal implementation of StarRocks sink connector is cached and imported by stream load in batches.
 
-### save_mode_create_template [string]
+## Sink Options
+
+| Name                         | Type    | Required | Default         | Description                                                                                                                                                                                                                  |
+|------------------------------|---------|----------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| nodeUrls                     | list    | yes      | -               | `StarRocks` cluster address, the format is `["fe_ip:fe_http_port", ...]`                                                                                                                                                     |
+| base-url                     | string  | yes      | -               | The JDBC URL like `jdbc:mysql://localhost:9030/` or `jdbc:mysql://localhost:9030` or `jdbc:mysql://localhost:9030/db`                                                                                                        |
+| username                     | string  | yes      | -               | `StarRocks` user username                                                                                                                                                                                                    |
+| password                     | string  | yes      | -               | `StarRocks` user password                                                                                                                                                                                                    |
+| database                     | string  | yes      | -               | The name of StarRocks database                                                                                                                                                                                               |
+| table                        | string  | no       | -               | The name of StarRocks table, If not set, the table name will be the name of the upstream table                                                                                                                               |
+| labelPrefix                  | string  | no       | -               | The prefix of StarRocks stream load label                                                                                                                                                                                    |
```

Comment:
From Doris Sink Connector I found use stream load label can implement 2PC and make sink support `exactly-once`. What does this look like in Starlocks?

Reply:
Currently, starrocks does not support exactly-once

Reply:
Ok